### ISSUE TITLE
 Declaration of pocketmine\level\Level::getSeed() must be compatible with pocketmine\level\ChunkManager::getSeed()

### ISSUE THREAD

**Issue by FusionDawg** [PocketMine/PocketMine-MP#3988]
I am using PocketMine-MP_1.6dev-11_068ad5e9_API-2.0.0.phar that was compiled on Feb 22, 2016.

I am getting the following message when trying to start the PocketMine-MP server.

````
[13:39:48] [Server thread/INFO]: Loading pocketmine.yml...
[13:39:48] [Server thread/INFO]: Loading server properties...
[13:39:48] [Server thread/INFO]: Selected English (eng) as the base language
[13:39:48] [Server thread/INFO]: Starting Minecraft: PE server version v0.14.0.0 alpha
[13:39:48] [Server thread/INFO]: Opening server on 0.0.0.0:19132
[13:39:48] [Server thread/INFO]: This server is running PocketMine-MP version 1.6dev "[REDACTED]" (API 2.0.0)
[13:39:48] [Server thread/INFO]: PocketMine-MP is distributed under the LGPL License
[13:39:49] [Server thread/INFO]: Loading recipes...
[13:39:49] [Server thread/NOTICE]: Level "OurWorld" not found

Fatal error: Declaration of pocketmine\level\Level::getSeed() must be compatible with pocketmine\level\ChunkManager::getSeed(): int in phar://B:/tools/PocketMine-MP/PocketMine-MP.phar/src/pocketmine/level/Level__32bit.php on line 117
[13:39:49] [Server thread/EMERGENCY]: An unrecoverable error has occurred and the server has crashed. Creating a crash dump
[13:39:49] [Server thread/EMERGENCY]: Please upload the "B:\tools\PocketMine-MP\CrashDump_Tue_Feb_23-13.39.49-ACT_2016.log" file to the Crash Archive and submit the link to the Bug Reporting page. Give as much info as you can.
bin\php\php.exe: Exit 1
````

Here is the crash dump

````
PocketMine-MP Crash Dump Tue Feb 23 13:39:49 ACT 2016

Error: Declaration of pocketmine\level\Level::getSeed() must be compatible with pocketmine\level\ChunkManager::getSeed(): int
File: /src/pocketmine/level/Level__32bit
Line: 117
Type: E_COMPILE_ERROR

Code:
[108] use pocketmine\utils\LevelException;
[109] use pocketmine\utils\Random;
[110] use pocketmine\utils\ReversePriorityQueue;
[111] 
[112] 
[113] 
[114] 
[115] 
[116] 
[117] class Level implements ChunkManager, Metadatable{
[118] 
[119] 	private static $levelIdCounter = 1;
[120] 	private static $chunkLoaderCounter = 1;
[121] 	public static $COMPRESSION_LEVEL = 8;
[122] 
[123] 
[124] 	const BLOCK_UPDATE_NORMAL = 1;
[125] 	const BLOCK_UPDATE_RANDOM = 2;
[126] 	const BLOCK_UPDATE_SCHEDULED = 3;
[127] 	const BLOCK_UPDATE_WEAK = 4;

Backtrace:
#0 (): pocketmine\Server->crashDump(boolean)

PocketMine-MP version: 1.6dev #0 [Protocol 45; API 2.0.0]
Git commit: 0000000000000000000000000000000000000000
uname -a: Windows NT SCORPIO 10.0 build 10586 (Windows 10) i586
PHP Version: 7.0.3
Zend version: 3.0.0
OS : WINNT, win
````

**Comment by justinnichols** [comment]
Same for me with same version of phar and PHP version.

**Comment by CyberPowerCrazii** [comment]
This has been resolved in latest commit. Close the issue.

**Comment by PEMapModder** [comment]
Already fixed in eb00dee

**Comment by FusionDawg** [comment]
Thanks for the fix.  I was looking for an updated phar file,  just realized that I can run PocketMine-MP from src (unpackaged).

Confirmed that it's working now.
